targets/zephyr/Makefile.zephyr

```diff
@@ -70,7 +70,7 @@ CONFIG_TOOLCHAIN_VARIANT = x86
 CPU = x86
 EXT_CFLAGS += -march=pentium
 EXT_CFLAGS += -mpreferred-stack-boundary=2 -mno-sse
-else ifeq ($(BOARD),qemu_cortex_m3)
+else ifeq ($(BOARD),$(filter $(BOARD),qemu_cortex_m3 frdm_k64f))
 CONFIG_TOOLCHAIN_VARIANT = arm
```

Comment:
> Might be worth to add a separate case for the FRDM-K64F board with more aggressive compiler flags

I agree that would be nice follow-up target. This one is however a small patch to add support of a  first physical ARM/Cortex-M board for Zephyr port (previously it was tested with just QEMU).

And when doing such CPU-specific optimizations, it would be nice to reuse machinery Zephyr already has for that, instead of building parallel if-else ladders in JerryScript makefile. I know that work on better integration with 3rd-party libs is ongoing in Zephyr, so another reason not to go for "premature optimization" at this spot. Hope that makes sense.

Thanks for review!